src/templates/_stripView.hbs

```diff
@@ -1,7 +1,20 @@
+<script src="assets/scripts/client/jquery.min.js"></script>
```

Comment:
This is a big no-no in my book.  Any scripts that need to be included should be included in the main bundle.  If, for some reason, that is not possible then a script should be included before the bundle at the foot of the `_layout.hbs` template.

Additionally, `jquery` is already included in the project as part of the bundle, so including it here like this will cause problems with eventual version conflicts.